flight/Modules/Stabilization/stabilization.c

```diff
@@ -324,7 +324,35 @@ static void stabilizationTask(void* parameters)
 
 					break;
 
-				case STABILIZATIONDESIRED_STABILIZATIONMODE_ATTITUDE:
+			case STABILIZATIONDESIRED_STABILIZATIONMODE_ACRO:
+					// this implementation is based on the Openpilot/Librepilot Acro+ flightmode
+					// and our existing rate & MWRate flightmodes
+					if(reinit)
+							pids[PID_RATE_ROLL + i].iAccumulator = 0;
+
+					// The unscaled input (-1,1)
+					float *raw_input = &stabDesired.Roll;
```

Comment:
It doesn't look like this will actually range from -1,1. If you look above you are scaling the input (which is -1,1) through an exponential by stabSettings.ManualRate